coursebuilder/modules/explorer/_static/js/course.js

```diff
@@ -20,13 +20,27 @@ $(document).ready(function documentReady() {
     var sidebarWidth = sidebarToggle ? $('.course-accordion').width() : 0;
 
     $('.course-accordion').toggleClass('hide-sidenav');
-    $('.course-card').animate({'margin-left': sidebarWidth / 2 + 'px'}, 300);
-    $('.course-accordion').animate({'margin-left': '-' + sidebarWidth + 'px'}, 300);
+    $('.course-card').animate({ 'margin-left': sidebarWidth / 2 + 'px' }, 300);
+    $('.course-accordion').animate({ 'margin-left': '-' + sidebarWidth + 'px' }, 300);
   });
 
   $('.course-accordion .collapsible').clone().appendTo('#courseMobileSidebar');
   $('.collapsible').collapsible();
   $('body').on('click', '#closeSideNav', function () {
     $('.button-collapse').sideNav('hide');
   });
+  
+  $('body').on('click', function (event) {
```

Comment:
@andela-johia is this too board given that we know where the carousel we are targeting is located?

Reply:
@andela-johia What is the user never clicks but expects to use the keyboard instead to navigate?

Reply:
@chegejames I am sorry for the misunderstanding what I meant to say was that until the user clicks within the carousel container before the arrow-key navigation is enabled.

Reply:
@chegejames pertaining to question about `line 33`, I used the `body` tag because I wanted the fact once the user clicks on the body in the page the code checks whether the `course-carousel` div was clicked on. I think we can narrow it down to using the `course-content` container div instead of the `body` tag. What do you think about that?